Libplanet/Store/Trie/Nodes/BaseNode.cs

```diff
@@ -0,0 +1,36 @@
+using System.Security.Cryptography;
+using Bencodex;
+using Bencodex.Types;
+
+namespace Libplanet.Store.Trie.Nodes
+{
+    internal abstract class BaseNode : INode
+    {
+        private static Codec _codec;
+
+        static BaseNode()
+        {
+            _codec = new Codec();
+        }
+
+        public BaseNode(INode value)
+        {
+            Value = value;
+        }
+
+        public HashDigest<SHA256> Hash
+        {
+            get
+            {
+                return Hashcash.Hash(Serialize());
+            }
+        }
```

Comment:
How about this?
```suggestion
        public HashDigest<SHA256> Hash => Hashcash.Hash(Serialize());
```

Reply:
Thanks for reviewing it, but it seems not used in anywhere because of `INodeExtensions.Hash()`. I will remove it instead. 🙏